tests/integration/test_github.py

```diff
@@ -6,6 +6,16 @@
 from ogr.abstract import PRStatus
 from ogr.services.github import GithubService
 
+def skip_tests(env_vars_list):
+    requirements_met = True
+    for item in env_vars_list:
+        if os.environ.get(item) is None:
+            requirements_met = False
```

Comment:
```python
  requirements_met = all(os.environ.get(item) for item in env_vars_list)
```

Reply:
yep, this could be used, it is not really same, same could be: replaced it with ``os.environ.get(item) is not None`` because ``A=""``  will be transformed to False, although it is defined.